src/components/snackbar/container.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Container to keep track of all snackbar elements that gets added to the page.
+ * When an element gets added or removed, the container will emit a
+ * `changeOffset` event on all elements in the container, letting them know
+ * the new offset to where they should position themselves.
+ */
+export class SnackbarContainer {
+    private snackbarElements: HTMLLimelSnackbarElement[] = [];
+
+    /**
+     * Add a new element to the container
+     *
+     * @param snackbar - element to add
+     */
+    public add(snackbar: HTMLLimelSnackbarElement) {
+        const popover = this.getPopover(snackbar);
+
+        // Stencil does not seem to recognise the existance of showPopover
+        // @ts-ignore
+        popover?.showPopover();
```

Comment:
😲

Yeah, the Stencil version we are using is over a year old, and `popover` is marked as "Baseline 2024 - Newly Available" on MDN:

<img width="767" alt="image" src="https://github.com/user-attachments/assets/afa1a59e-45bf-482b-bc4c-afe54926238f">

Myself, I went looking for how we define `showPopover` on the element, and after not finding _anything_, I finally googled for "js popover", and then I went 🤯… 😂

Reply:
Yeah it would be nice to use this for `limel-popover` and `limel-tooltip` as well in the future, but I think we need anchor positioning to be baseline as well first 
https://developer.chrome.com/blog/anchor-positioning-api

Reply:
and limel-modal before all of the above. that one doesn't even use portal

Reply:
Yeah the dialog we can do already today :smiley:

Reply:
let me know if you wanna start on it ;)